Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/IncidentConsumer/IncidentConsumer.csproj

```diff
@@ -10,13 +10,17 @@
         <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="5.3.0" />
         <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.4.0" />
         <PackageReference Include="Azure.Identity" Version="1.8.0" />
+        <PackageReference Include="Oocx.ReadX509CertificateFromPem" Version="1.2.3" />
```

Comment:
You probably don't need the Certificate library because you don't use it anymore

Reply:
fixed